apps/script/HoC2023AiGenerateWeights.py

```diff
@@ -107,33 +111,83 @@
 background_dict = {}
 foreground_dict = {}
 
-# Calculate and print similarity scores
-for emoji_name in emoji_names_list:
-    palette_scores = []
-    for palette_word in palette_model_descriptive_names:
-        id_token = nlp(emoji_name)
-        palette_token = nlp(palette_word)
-        similarity_score = id_token.similarity(palette_token)
-        palette_scores.append(round(similarity_score, 2))
+# Define caches to store ada's raw embedding outputs to reduce query costs
```

Comment:
Could you define what raw embedding outputs are or include a reference like https://platform.openai.com/docs/guides/embeddings/what-are-embeddings?